app/config/services.yml

```diff
@@ -31,9 +31,15 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest }
 
-    # Event subscribers are similar to event listeners but they don't need to add
-    # a separate tag for each listened event. Instead, the PHP class of the event
-    # subscriber includes a method that returns the list of listened events.
+    app.comment_notification:
+        class: AppBundle\EventListener\CommentNotificationListener
+        arguments: ['@mailer', '@router', '@translator', '%app.notifications.email_sender%']
```

Comment:
OK, it is already part of the codebase although not used for notifications today